packages/ui/src/apps/collectionsUi/viewModules/start/Component.js

```diff
@@ -1,126 +1,128 @@
-                    An international partnership for open-source development
-                  </Header>
-                  <p style={{ fontSize: '1.33em' }}>
-                    {`DINA is developed by the DINA consortium, an unincorporated
-                  international partnership among organizations and individuals
-                  for collaborative open-source development. The DINA consortium
-                  was founded in 2014 by six natural history collection
-                  institutions in Europe and North America and is open to
-                  additional members as detailed below. The DINA acronym stands
-                  for "DIgital Information system for NAtural history data", and
-                  has its roots in a Swedish initiative to replace a
-                  heterogeneous collection of unsustainable in-house databases
-                  with a modern, web-based national collection management
-                  system.`}
-                  </p>
-                </Segment>
-              </Grid.Column>
-              <Grid.Column floated="right" width={6}>
-                <Segment padded="very">
-                  <Markdown
-                    fallbackLanguage="en"
-                    textKey="modules.start.changelog"
-                  />
-                </Segment>
-              </Grid.Column>
-            </Grid.Row>
+        <div className="ui center aligned grid">
+          <Grid
+            columns={2}
+            style={{ marginTop: '6.25em', maxWidth: '60em' }}
+            verticalAlign="middle"
+          >
+            <Grid.Column width={10}>
+              <Header
+                as="h1"
+                content={`${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'welcome',
+                })} ${capitalizeFirstLetterOfEachWord(
+                  username
+                )}! ${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'startPageText',
+                })}`}
+                style={{
+                  color: '#000000a0',
+                  fontWeight: 200,
+                  lineHeight: '2.5rem',
+                }}
+              />
+              <div
+                style={{
+                  marginBottom: '1em',
+                  marginTop: '2em',
+                }}
+              >
+                <NavLink to="/app/specimens/mammals/create/sections/0">
+                  <Button primary size="huge" type="button">
+                    <Icon name="plus" />
+                    {moduleTranslate({
+                      capitalize: true,
+                      module: 'start',
+                      textKey: 'btnRegisterMammal',
+                    })}
+                  </Button>
+                </NavLink>
+                <NavLink to="/app/specimens/mammals">
+                  <Button basic color="blue" size="huge" type="button">
```

Comment:
instead of `basic color="blue"` I think you should use `basic primary`, at least I guess this color should change if we change the color of `primary` buttons